src/modules/clark/learning-object-module/change-authorship.routes.ts

```diff
@@ -0,0 +1,10 @@
+import { HTTPMethod } from "../../../shared/types/http-method.type";
+import { ProxyRoute } from "../../../shared/types/proxy-route.type";
+
+export const CHANGE_AUTHORSHIP_ROUTES: ProxyRoute[] = [
+    {
+        method: HTTPMethod.POST,
+        path: "/users/:userId/learning-objects/:learningObjectId/change-author",
```

Comment:
This didn't get switched to CUID? I haven't had a chance to look.

Reply:
No according to the API
![image](https://github.com/user-attachments/assets/0ecff29c-b71a-4eb6-928a-032676a64619)